fe/fe-core/src/main/java/com/starrocks/qe/AuditLogBuilder.java

```diff
@@ -121,26 +127,42 @@ public void exec(AuditEvent event) {
                         continue;
                     }
                 }
-                sb.append("|").append(af.value()).append("=").append(value);
+
+                if (Config.audit_log_json_format) {
+                    logMap.put(af.value(), value);
+                } else {
+                    sb.append("|").append(af.value()).append("=").append(value);
+                }
             }
 
-            String auditLog = sb.toString();
+            ObjectMapper objectMapper = new ObjectMapper();
+
             if (event.type == EventType.CONNECTION) {
-                AuditLog.getConnectionAudit().log(auditLog);
-            } else {
-                AuditLog.getQueryAudit().log(auditLog);
-                // slow query
-                if (queryTime > Config.qe_slow_log_ms) {
-                    AuditLog.getSlowAudit().log(auditLog);
+                if (Config.audit_log_json_format) {
+                    AuditLog.getConnectionAudit().log(objectMapper.writeValueAsString(logMap));
+                } else {
+                    AuditLog.getConnectionAudit().log(sb.toString());
                 }
 
+            } else {
                 if (isBigQuery(event)) {
-                    sb.append("|bigQueryLogCPUSecondThreshold=").append(event.bigQueryLogCPUSecondThreshold);
-                    sb.append("|bigQueryLogScanBytesThreshold=").append(event.bigQueryLogScanBytesThreshold);
-                    sb.append("|bigQueryLogScanRowsThreshold=").append(event.bigQueryLogScanRowsThreshold);
-                    String bigQueryLog = sb.toString();
-                    AuditLog.getBigQueryAudit().log(bigQueryLog);
```

Comment:
why remove slow audit log and big query audit log?

Reply:
I discussed this with @chaoyli earlier. We both agreed that the definitions of slow and big are ambiguous and their criteria can vary across different scenarios. Therefore, we believe that using the regular audit log is sufficient. Consumers can then decide how they want to filter the logs based on their specific needs.

Reply:
Directly removing the slow and big query logs is not user-friendly for the community, as it will bring them additional costs, because not every user has a dedicated tool to consume logs for filtering.
It seems that your pain point is that the three log files are redundant and ambiguous.
If sr provide a switch that allows you to turn off the slow and big query logs, can it also solve your problem? @dengliu

Reply:
> If sr provide a switch that allows you to turn off the slow and big query logs, can it also solve your problem?

Yes, it solve the problem. Can you make sure having the logs in json format. That's the main purpose of this PR.
Thanks a lot!